.pipelines/scripts/wait-for-resource.sh

```diff
@@ -0,0 +1,19 @@
+available=false
+max_retries=${MAX_RETRIES:-60}
+sleep_seconds=${SLEEP_SECONDS:-10}
+for ((i = 0; i < ${max_retries}; i++))
+do
+  if [[ ! $(kubectl wait --for=condition=available ${RESOURCE} --all --namespace ${NAMESPACE}) ]]; then
```

Comment:
The osm resources sometimes take a while to show up after the osm extension has been added. So I need to run this in a loop, otherwise if you call kubectl wait before the resources can be detected it will just exit the pipeline with an error.